styles/shop.module.scss

```diff
@@ -1,10 +1,36 @@
-// .root {
-//   min-height: 100vh;
-//   flex: 1;
-//   padding: 4rem 2rem;
-//   display: flex;
-//   flex-direction: column;
-//   justify-content: center;
-//   align-items: center;
-//   overflow-x: hidden;
-// }
+.header_content {
+  display: flex;
+  flex-direction: row;
+  max-width: 900px;
+  gap: 3rem;
+  margin: 4rem 0;
+  .header_text_container {
+    display: flex;
+    flex-direction: column;
+    .header_title {
+      font-size: 2rem;
+      font-weight: 400;
+      margin: 2rem 0 0 0;
+    }
+    .header_text {
+      font-size: 1.75rem;
+      width: 710px;
+      a {
+        color: var(--link-color);
+      }
+    }
+  }
+}
+
```

Comment:
Why have you moved the CSS from global to here? and removed one of the `@media`? Now it doesn't resize properly in resolutions between 1200px and 800px